plugins/node/opentelemetry-instrumentation-mysql/src/instrumentation.ts

```diff
@@ -328,7 +337,9 @@ export class MySQLInstrumentation extends InstrumentationBase<
           });
         }
         span.end();
-        return originalCallback(...arguments);
+        return context.with(parentContext, () =>
```

Comment:
:point_up:

Reply:
Both cases produce compiler errors, as the original function is of `Function` type. In the first case it's an overloaded function so unless there's a good/simple way to fix the types I'd leave it as is :thinking:

Reply:
Oh well! Let's not dwell on that too much then!